### PR TITLE
docs: openstack documentation preparation fixes

### PR DESCRIPTION
**Pull request by v1k0d3n**
**What is the purpose of this pull request?**: Fix documentation broken links

**What issue does this pull request address?**: Fixes ##323 

**Notes for reviewers to consider**: @renmak had some comments on this, @larryrensing noticed it originally.

**Specific reviewers for pull request**: @renmak @larryrensing

### CODE REVIEW

File: docs/guides-install/third-party-tools/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Third-Party Installation Tools
+
+- [Armada](armada.md) - Overview of Helm-Toolkit
```
Comment (wilkers-steve) [inline docs/guides-install/third-party-tools/readme.md]: Minor oversight -- just needs a different description here

File: docs/guides-operator/ops-kubernetes/readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Openstack-Helm Operations: Kubernetes
+
+## Conceptual Guides: Kubernetes
+- [Init-Containers](kb-init-containers.md)
+- [Jobs](kb-jobss.md)
```
Comment (wilkers-steve) [inline docs/guides-operator/ops-kubernetes/readme.md]: s/jobss/jobs

File: docs/guides-operator/readme.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Openstack-Helm: Operations Guides
+- [Helm Operations](ops-helm/readme.md) - Helm Operator Guides
+  - [Openstack-Helm Operations](ops-helm/osh-operations.md)
+  - [Addons and Plugins](ops-helm/osh-addons.md)
+- [Kubernetes Operations](ops-kubernetes/readme.md)
+  - [Init-Containers](ops-kubernetes/kb-init-containers.md)
+  - [Jobs](ops-kubernetes/kb-jobs.md)
+- [Openstack Operations](readme.md)
```
Comment (wilkers-steve) [inline docs/guides-operator/readme.md]: different filepath here.  im assuming this would be `ops-openstack/readme.md`

**Comment by v1k0d3n** [comment]
completed changes @wilkers-steve made.

**Pull request merged**